community-operators/microcks/microcks-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: microcks-operator.v0.1.0
+  name: microcks-operator.v0.2.0
```

Comment:
In order for users to upgrade using this change, they will need to uninstall the old version and install the new version. For OLM to support automatic upgrades for this operator, you will need to make a new CSV for the new version, with the `spec.replaces` field set to the name of the old CSV

Reply:
Now done in commit ad6b91079dbb6f04325865ca4ca1208c49e9a993. Was not aware of this practice. Thanks!